### DISCUSSION TITLE
Duda link de repositorio

### DISCUSSION THREAD

**Discussion by alb333z** [LaunchX-InnovaccionVirtual/CursoIntroPython#169] [General]
Buenas noches fer @FernandaOchoa disculpa si mi usuario de github no tiene nada que ver con mi nombre por ejemplo mi nombre de usuario es alb333z mi nombre es AlbertoAviña tendria que crear otra cuenta de github? ya llevaba un avance. O existe alguna forma de renombrar mi usuario..

![image](https://user-images.githubusercontent.com/48806691/153694537-7fbe1f9e-4ccf-48e9-bced-9eff4af5f17b.png)

**Comment by OscarPedroza** [comment]
Creo que no importa el nombre del usuario, tienen un sistema que comprueba que ese usuario este en la organización y eso lo validará, además en el formulario introduces el nombre completo.

**Reply by alb333z** [reply]
Gracias Oscar, ya estaba un poco preocupado.

**Comment by Nukario** [comment]
No debería haber problema, si puedes cambiar el nombre de usuario pero puede causar muchos errores en donde este vinculado (o eso dice github) con que estés dentro de la organización es suficiente igual en el formulario tienes que poner tu nombre

**Reply by alb333z** [reply]
Muchas gracias, si de echo me registre con esa cuenta de github a la organización.  Muchas gracias se los agradezco

**Comment by LuisSantiago10** [comment]
No hay problema, pero Fernanda comentó que si podrias tener un mejor nombre de usuario lo colocarás ya que al final este será tu cv como programador. Saludos

**Reply by alb333z** [reply]
Tienes razón Luis, esa cuenta ya tenia tiempo que la había creado, la verdad para ser sincero no sabia nada de github cuando la cree, por eso no tome en cuenta el nombre de usuario :( pero si recuerdo que Fer comento acerca del nombre, creo que tendré que crear otra cuenta con mi nombre completo de usuario, muchas gracias.

**Comment by AlejandroMontoyaT** [comment]
En el link donde te enseñan a crear tu cuenta de github, recomienda. Cómo debe de ir tu nombre. Por qué tu github es tu carta de presentación. 
![Screenshot_20220211_212149.jpg](https://user-images.githubusercontent.com/83684130/153694842-784255b4-c0f6-4180-a4b5-54d258cb6a2c.jpg)

**Reply by alb333z** [reply]
Gracias alex, si vi la información, lo que pasa es que quise reutilizar esa cuenta de github que ya tenia de hace algún tiempo.

**Reply by DanielChavezZ** [reply]
me parece que te lo detecta en automatico si estas en la organizacion

**Reply by AlejandroMontoyaT** [reply]
Ok, igual tenía mi cuenta de tiempo atrás pero cambié el name.

**Reply by alb333z** [reply]
Me podrías decir como le hiciste por favor Alex. busque en edit profile pero solo me aparece el name y lo cambie, pero creo que ese no hace referencia al nombre de usuario.
![image](https://user-images.githubusercontent.com/48806691/153695284-1ed93ea9-42e2-4652-840c-eed989204c51.png)

**Reply by AlejandroMontoyaT** [reply]
En ajustes de tu cuenta y busca está opción. CHANGE USERNAME
![Screenshot_20220211_214412.jpg](https://user-images.githubusercontent.com/83684130/153695446-78a238bc-a8cd-49e7-b87a-56414ab6d3d5.jpg)

**Reply by alb333z** [reply]
Muchas gracias alex
